src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Results/_ResultsPage.cshtml

```diff
@@ -50,21 +50,16 @@
 
     <div class="row">
         <div class="col-xs-12">
-            <a asp-controller="Results"
-               asp-action="Index"
-               asp-route-pageIndex="@(entries.PageIndex - 1)"
+            <a id="prevPage" href="#week1" onclick="GetResults(@weekNumber, @(entries.PageIndex - 1))"
```

Comment:
why _"#week1"_? Maybe "#ranking"?

Reply:
fixed that, because it wasn't allowing to autoscroll the page